### PR TITLE
Added \mathbb

### PR DESCRIPTION
**Pull request by michaelniedermayr**
## Description

Added \mathbb{}.

## Checklist

- [x] I added a PR description.
- [x] I linked all related issues I could find.
- [x] If this PR changes anything about the main `catex` or `example` package (also README etc.),
      I created an entry in `CHANGELOG.md` (`## NEXT RELEASE` if it is not worth an update).
- [x] If this PR includes a notable change in the `catex` package, I updated the version according
      to [Dart's semantic versioning](https://dart.dev/tools/pub/versioning#semantic-versions).
- [x] If this PR adds new functions, I updated the `function_prioritization.csv` file.
- [x] I added tests covering all my additions (Golden tests for anything rendering related).
- [x] All required checks pass.

### CODE REVIEW

File: lib/src/lookup/functions.dart [hunk 1]
```diff
@@ -207,6 +212,7 @@ const List<CaTeXFunction> supportedTextFunctions = [
   CaTeXFunction.textMd,
   CaTeXFunction.textIt,
   CaTeXFunction.textUp,
+  CaTeXFunction.mathbb,
```
Comment (creativecreatorormaybenot) [inline lib/src/lookup/functions.dart:215]:
```suggestion
```

The function generally does not seem to be supported in text mode.

File: pubspec.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: catex
 description: >-
   Fast math TeX renderer for Flutter written in Dart. Outputs TeX formulas (like LaTeX,
   KaTeX, MathJax, etc.) inline using only Flutter.
-version: 0.0.1+2
+version: 0.0.1+3
```
Comment (creativecreatorormaybenot) [inline pubspec.yaml:5]: Depending on whether this is merged before or after #50, I would remove the bump here and bump it there, but you do not need to change anything about that 👍🏽

**Comment by creativecreatorormaybenot** [comment]
@michaelniedermayr It looks like you need to amend your commit in order to add **an** email. Note that you can add the [GitHub no-reply email](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address) as well if you prefer that 🙃

**Comment by creativecreatorormaybenot** [comment]
@michaelniedermayr Awesome 👌🏽 

Looks like you need to put your username into the form the CLA bot linked 😬

**Comment by michaelniedermayr** [comment]
Just filled out the form. Hope that this time everything is correct.

**Comment by creativecreatorormaybenot** [comment]
@cla-bot check

**Comment by michaelniedermayr** [comment]
@cla-bot check

**Comment by creativecreatorormaybenot** [comment]
Yep, very nice 🙌🏽 Next time, the cla thing is already set up 😄

Will be published in #50 - only waiting for a review 🙂

**Pull request merged**